client/a8c-for-agencies/sections/purchases/payment-methods/payment-method-overview/stored-credit-card-delete-dialog/delete-primary-confirmation/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { PaymentLogo } from '@automattic/wpcom-checkout';
+import { useTranslate } from 'i18n-calypso';
+import type { PaymentMethodCard } from 'calypso/jetpack-cloud/sections/partner-portal/payment-methods';
+
+import './style.scss';
+
+interface Props {
+	card: PaymentMethodCard;
+	altCard?: PaymentMethodCard;
+	isFetching: boolean;
+}
+
+const DeletePrimaryCardConfirmation = ( { card, altCard, isFetching }: Props ) => {
+	const translate = useTranslate();
+
+	if ( isFetching ) {
+		return (
+			<div className="delete-primary-card-confirmation">
+				<div className="delete-primary-card-confirmation__card-details-loader" />
+			</div>
+		);
+	}
+
+	if ( ! altCard ) {
+		return (
+			<div className="delete-primary-card-confirmation">
+				<div className="delete-primary-card-confirmation__card">
+					<p className="delete-primary-card-confirmation__notice">
+						{ translate(
+							'Issuing new licenses will be paused until you add a new primary payment method. ' +
+								'Additionally, the existing licenses will be revoked at the end of their respective terms.'
+						) }
```

Comment:
I'm curious about this `translate`  usage and if the translation system takes the whole text with the `+` concantation.

Reply:
I will probably split this string into two `translate` calls. The linting was auto-fixing this. Thanks for bringing this to my attention.

Reply:
I lied 😀; this was intentional from the original source file, and I assumed it was the linting auto-fixing. This has been fixed.

Reply:
So the answer is yes, `translate` (and translators) manage this slip well... Or that is what we guess :)

Reply:
Yes, the system seems to recognize it. But for good practice(I think), this should be set into one string.